src/span.h

```diff
@@ -57,4 +59,13 @@ constexpr Span<A> MakeSpan(A (&a)[N]) { return Span<A>(a, N); }
 template<typename V>
 constexpr Span<typename std::remove_pointer<decltype(std::declval<V>().data())>::type> MakeSpan(V& v) { return Span<typename std::remove_pointer<decltype(std::declval<V>().data())>::type>(v.data(), v.size()); }
 
+/** Pop the last element off a span, and return a reference to that element. */
+template<typename T>
```

Comment:
nit, missing space(git-clang-format)
```suggestion
template <typename T>
```

Reply:
Ok.